tests/AgendaTest.php

```diff
@@ -84,4 +84,32 @@ public function testTenBallotsWithTies() : void
         $instance = new Agenda(...$ballots);
         $this->assertOnlyContainsCandidates($instance->getCandidates(), array($this->alice, $this->bob, $this->claire));
     }
```

Comment:
I'm not sure why these tests are included in the pull request. I'm pretty sure they were merged a while ago. Rebase craziness?